sov-modules/sov-modules-api/src/lib.rs

```diff
@@ -129,7 +115,7 @@ pub trait Spec {
         + Eq
         + TryFrom<&'static str>
         + Clone
-        + Debug
+        + fmt::Debug
```

Comment:
Why?

Reply:
It saved one `#[cfg(feature = "native")]` in the imports as `fmt::Display` is use only with `native` feature. So I can use `use std::fmt` for both `native` and `not(native)`.

Reply:
fixed